packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -840,26 +839,27 @@ ensureIndex(Posts,
 );
 
 Posts.addView("nearbyEvents", (terms: PostsViewTerms) => {
-  const yesterday = moment().subtract(1, 'days').toDate();
   const onlineEvent = terms.onlineEvent === false ? false : viewFieldAllowAny
   let query: any = {
     selector: {
       location: {$exists: true},
       groupId: null,
       isEvent: true,
       onlineEvent: onlineEvent,
-      $or: [{startTime: {$exists: false}}, {startTime: {$gt: yesterday}}],
+      $or: [{startTime: {$exists: false}}, {startTime: {$gt: new Date()}}, {endTime: {$gt: new Date()}}],
       mongoLocation: {
         $near: {
           $geometry: {
                type: "Point" ,
                coordinates: [ terms.lng, terms.lat ]
```

Comment:
It'd be better if it's easy to remove distance sorting if the user doesn't have a location